2-mandatory/4-restaurant.js

```diff
@@ -20,9 +20,21 @@ const MENU = {
 };
 
 let cashRegister = {
-  // write code here
-}
-
+  orderBurger: function (balanceNumber) {
+    let myBalance = balanceNumber;
+    if (myBalance >= 6.5) {
+      myBalance = myBalance - 6.5;
+    }
+    return myBalance;
```

Comment:
good job, here you should use MENU.burger instead of using 6.5, and you do not need to create a new variable for putting your parameters in new variable, you can use parameters straight away.